README.md

```diff
@@ -10,7 +10,48 @@ The Message Format Working Group (MFWG) is tasked with developing an industry st
 - [Goals and Non-Goals](docs/goals.md)
 - [Record of Consensus Decisions](docs/consensus_decisions.md)
 
-## MessageFormat 2 Draft Syntax
+## MessageFormat 2 Technical Preview
+
+The MessageFormat 2 specification was accepted by the CLDR-TC on 2024-02-28
+as a new part of the LDML specification.
```

Comment:
We care about the mechanics here, but the readers of this probably don't. Also, linkify.

```suggestion
The MessageFormat 2 specification is a new part of
the [LDML](https://www.unicode.org/reports/tr35/) specification.
```